go.mod

```diff
@@ -11,10 +11,11 @@ require (
 	github.com/redhat-cop/operator-utils v0.3.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
-	k8s.io/api v0.18.2
+	k8s.io/api v0.18.6
```

Comment:
were these intended or just automated?

Reply:
I was trying to fix an issue, this should be irrelevant, but I can revert it.